ansible_rulebook/cli.py

```diff
@@ -186,6 +193,8 @@ def main(args: List[str] = None) -> int:
         if args.controller_token:
             job_template_runner.host = args.controller_url
             job_template_runner.token = args.controller_token
+            if args.controller_no_ssl_verify:
```

Comment:
@bzwei it might make more sense to use a single variable called controller_ssl_verify with 3 values
- yes
- no
- provide a ca certificate bundle

https://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification

We had coded this in catalog like here https://github.com/ansible/pinakes/blob/122fc3184b0c6b6745de220bbd42e632531ec93b/pinakes/main/inventory/task_utils/tower_api.py#L28

Reply:
Not sure if the application should handle custom certs. awx does it?

pinging @jamesmarshall24 for feedback.